synapse/lib/agenda.py

```diff
@@ -707,7 +707,12 @@ async def clearRunningStatus(self):
         for appt in list(self.appts.values()):
             if appt.isrunning:
                 logger.debug(f'Clearing the isrunning flag for {appt.iden}')
-                await self.core.addCronEdits(appt.iden, {'isrunning': False})
+                edits = {
+                    'isrunning': False,
+                    'lastfinishtime': self._getNowTick(),
+                    'lasterrs': ['aborted'] + appt.lasterrs[-4:]
```

Comment:
Maybe something more descriptive like `cancelled due to cortex event`?

Reply:
I don't think that's much more descriptive. Since we can't infer what happened, `aborted` seems fine with me :+1: